### ISSUE TITLE
improper exit code when running against a directory

### ISSUE THREAD

**Issue by travi** [syngan/vim-vimlint#83]
When running the `bin/vimlint.sh` script against a single file, an exit code of `2` is returned when the file contains lint. However, a `0` exit code is returned when running the script against a directory unless the last (alphabetically) file has lint, even if a file that is not the last one does have lint. 

In order for this to be an effective tool for continuous integration, properly returning a non-zero exit code is important if any lint is present.

**Comment by syngan** [comment]
Could you tell me how to reproduce the issue? I just tried, but it seems to work as expected.

**Comment by travi** [comment]
Of course, now that I tried it again, its working properly for me as well. I think I'm running everything the same way and don't remember updating anything either.

The previous behavior that I saw listed all files in the directory, but also listed errors for the file that had them. Now I'm seeing output only for the file that the error and color is added the the `Error` that wasn't there before.

I don't have any explanation for the difference in behavior at this point, so I'll close this issue. If I run across the odd behavior again, I'll try to narrow down the details and re-open.
